src/Clearinghouse.sol

```diff
@@ -250,7 +250,17 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
     /// @param *unused loadID_ of the load.
     /// @param amount_ repaid (in DAI).
     function _onRepay(uint256, uint256 amount_) internal override {
-        _sweepIntoDSR(amount_);
+        if (active) {
+            _sweepIntoDSR(amount_);
+        } else {
+            uint256 outstandingDebt = TRSRY.reserveDebt(dai, address(this));
```

Comment:
Can instead call `TRSRY.repayDebt` instead of this entire block

Reply:
if the CH is active we don't repay debt, as funds stay there available for other users to take loans.
I have simplified the code to call `_defund()` if CH is deactivated instead.